src/py42/services/fileevent.py

```diff
@@ -23,7 +23,7 @@ def search(self, query):
         """
         query = str(query)
         uri = u"/forensic-search/queryservice/api/v1/fileevent"
-        return self._session.post(uri, data=query)
+        return self._connection.post(uri, data=query)
 
     def get_file_location_detail_by_sha256(self, hash):
```

Comment:
Overriding a built-in here

Reply:
assuming you mean `hash`. This wasn't changed as part of this PR and is technically a breaking change (in the case that a keyword argument is used), but I'm still willing to change it to `checksum` or something